bundle/cert-manager-operator/0.15.0-alpha.0/cert-manager-operator.v0.15.0-alpha.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    description: "x509 certificate management for Kubernetes"
+    createdAt: "2020-04-20T12:00:00Z"
+    support: "cert-manager team"
+    alm-examples: ' [{"apiVersion":"operator.cert-manager.io/v1alpha1","kind":"CertManager","metadata":{"name":"cert-manager","namespace":"cert-manager"},"spec":{}}]'
+    capabilities: Full Lifecycle
+    categories: Security
+    certified: 'false'
```

Comment:
Presumably we flip this somewhere further down the line?

Reply:
We should, for now it isn't fully certified yet